### ISSUE TITLE
OnTheBlock and NextDoor join forces to finally finally finally make digital identities that don't suck

### ISSUE THREAD

**Issue by humbitious** [ConsenSys/web3studio-projects#3]
## The Story

NextDoor has an interesting attribute...people in your neighborhood have to confirm that a new NextDoor signup is indeed living and known to people in their neighborhood. Sound like the most essential human attestation ever? Yup! But it needs a little Web3 love to fix some incentive and game problems and to turn it into the global identity powerhouse that it can be.

**Comment by brian-lc** [comment]
I agree that the NextDoor sign-up is pretty slick (in that they physically mail you a code at your address to validate your account). Where you live is a part of your identity but not all of it and not forever (or even a few years if you rent). I think this would be interesting if you somehow made it about local news and reporting. Like CIVL but for your neighborhood/town/county. Funding local news by locals for locals.
